### DISCUSSION TITLE
Linux (Sh1mmer)

### DISCUSSION THREAD

**Discussion by kunai4K** [3kh0/ext-remover#547] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
What cool things can I do with Linux after unenrolling?

**Comment by obkduishinijf** [comment, marked as answer]
you can use .exe files, and much more. it's basically windows, but good, and for chromebooks.

**Reply by BroJac5246** [reply]
It's complicated, though. I've never succeeded in using WINE (the program needed to run Windows programs) on Crostini. Also, be aware that there's no GUI; it's just a terminal when you download Linux.

**Reply by amelia-bartlett** [reply]
> you can use .exe files, and much more. it's basically windows, but good, and for chromebooks.

As a Linux user, this physically hurts.

Firstly - you don't run .exe files.  (there are compatibility layers such as WINE or Proton, but they don't always work)
Secondly - its not "basically windows".  Its far superior
Thirdly - its not just for chromebooks.  I am typing this from my custom-built pc running Linux (Debian 11, if you are interested).

**Reply by amelia-bartlett** [reply]
> It's complicated, though. I've never succeeded in using WINE (the program needed to run Windows programs) on Crostini. Also, be aware that there's no GUI; it's just a terminal when you download Linux.

GUI apps do work if you install them (crostini uses apt with the Debian 11 repos)

**Reply by BroJac5246** [reply]
> > It's complicated, though. I've never succeeded in using WINE (the program needed to run Windows programs) on Crostini. Also, be aware that there's no GUI; it's just a terminal when you download Linux.
> 
> GUI apps do work if you install them (crostini uses apt with the Debian 11 repos)

Yes, but at the beginning there are none.
